geode-core/src/main/java/org/apache/geode/cache/client/internal/ProxyRegion.java

```diff
@@ -388,7 +388,7 @@ public Set keySetOnServer() {
   public Set keys() {
```

Comment:
I think you want this old "keys()" implementation to now be "keySet()". The old "keySet" did not properly call preOp/postOp.